src/test/java/org/folio/logintest/LogEventsApiTest.java

```diff
@@ -113,6 +111,14 @@ public void setUp(TestContext context) {
     });
   }
 
+  @Test
+  public void testMissingOkapiUrl() {
+    JsonObject logEven = getLogEven("tenant", UUID.randomUUID().toString(), LogEvent.EventType.PASSWORD_RESET.toString());
```

Comment:
This exists before but can you rename `getLogEven` to `getLogEvent`?

Reply:
Sure

Reply:
Sure